hbase_sql/src/main/java/com/splicemachine/olap/OlapServerSubmitter.java

```diff
@@ -95,15 +95,17 @@
 
     private static final Logger LOG = Logger.getLogger(OlapServerSubmitter.class);
     private final ServerName serverName;
+    private final String queueName;
```

Comment:
nit: it's either `queue` or `queueName`. Could we agree on one way to refer to a queue?